src/components/Api/Api.jsx

```diff
@@ -0,0 +1,295 @@
+        if (res.ok) {
+          window.location.reload(false);
+        } else {
+          throw Error();
+        }
+      })
+      .catch(() => this.setState({ error: true }));
+  }
+
+  handleUpdate(token) {
+    this.setState({ loading: true });
+    fetch(`${process.env.REACT_APP_API_HOST}${URI}`, {
+      credentials: 'include',
+      method: 'PUT',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        token,
+      }),
+    })
+      .then((res) => {
+        if (res.ok) {
+          window.location.reload(false);
+        } else {
+          throw Error(strings.api_error);
+        }
+      })
+      .catch(() => this.setState({ error: true }));
+  }
+
+  render() {
+    const { loading, user } = this.props;
+    const showLoginButton = !user;
+    const showGetKeyButton = user && !(this.state.customer && this.state.customer.api_key);
+
+    return (
+      <div>
+        <Helmet title={strings.title_api} />
+        <ApiContainer style={{ textAlign: 'center' }}>
+          {
+            this.state.error ?
+              <div>
+                {strings.api_error}
+              </div>
+            : <div />
+          }
+          <h1>{strings.api_title}</h1>
+          <h3>{strings.api_subtitle}</h3>
+          {
+            loading || this.state.loading ?
+              <CircularProgress mode="indeterminate" />
+            :
+              <div>
+                { showLoginButton ?
+                  <a href={`${process.env.REACT_APP_API_HOST}/login`}>
+                    <RaisedButton primary label={strings.api_login} style={{ margin: '5px 5px' }} />
+                  </a>
+                  : <div />
+                }
+                { showGetKeyButton ?
+                  <StripeCheckout
+                    name="OpenDota"
+                    description={strings.api_title}
+                    billingAddress
+                    stripeKey={process.env.REACT_APP_STRIPE_PUBLIC_KEY}
+                    token={this.handleSubmit}
+                    zipCode
+                    locale="auto"
+                  >
+                    <RaisedButton primary label={strings.api_get_key} style={{ margin: '5px 5px' }} />
+                  </StripeCheckout>
+                  : <span />
+                }
+                <a href="//docs.opendota.com">
+                  <RaisedButton label={strings.api_docs} style={{ margin: '5px 5px' }} />
+                </a>
+                { this.state.customer ?
+                  <div>
+                    { this.state.customer.api_key ?
+                      <div>
+                        <h4>{strings.api_header_key}</h4>
+                        <KeyContainer>{this.state.customer.api_key}</KeyContainer>
+                        <p>{`${strings.api_billing_cycle
+                            .replace('$date', moment.unix(this.state.customer.current_period_end).format('MM-DD-YYYY'))} ${
```

Comment:
you should be able to find a standalone implementation of unix() and use toDateString() to format.